src/Assignment4/NothingFoundComponent/index.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+import userIcon from "../../logo/userIcon.png";
+import "../CardComponent/styles-card.css";
+
+const CardComponent = ({}) => (
```

Comment:
you can remove the empty curly braces {}

Reply:
removed